<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlipViewPoint</class>
 <widget class="qMRMLWidget" name="FlipViewPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>207</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLNodeComboBox" name="sliceNodeSelector">
     <property name="toolTip">
      <string>Select a slice view</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLSliceNode</string>
      </stringlist>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="flipPushButton">
     <property name="toolTip">
      <string>Flip the selected view</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Flip view</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="restorePushButton">
     <property name="text">
      <string>Restore all views</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>FlipViewPoint</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>sliceNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
